src/components/CodeArea/CodeArea.styles.tsx

```diff
@@ -24,6 +24,14 @@ const ContentStyled = styled.div`
     display: flex;
 `;
 
+const MaximizeButtonStyled = styled.button`
```

Comment:
hey @DanielNgozika did you try to import Button instead of making another?

Reply:
All yours.